client/luigi-client.js

```diff
@@ -79,6 +83,10 @@
       if (window._init) {
         window._init(currentContext.context);
       }
+
```

Comment:
Maybe we could save us the roundtrip and hide the loading indicator automatically from core.